tools/torchci/clickhouse_query_perf.py

```diff
@@ -77,7 +77,11 @@ def get_avg_stats(query_ids: list) -> tuple:
 def get_query_ids(query: str, params: dict, times: int) -> list[str]:
     def _get_query_id(query: str, params: dict) -> Optional[str]:
         try:
-            res = get_clickhouse_client().query(query, params)
+            res = get_clickhouse_client().query(
+                query,
+                params,
+                settings={"enable_filesystem_cache": 0, "use_query_cache": 0},
```

Comment:
fyi, to reduce noise the CH folks suggested running a query 10 times and excluding the top 20% percentile perf scores to get your stable values

Reply:
This might be useful to have a parameter on the script to choose different statistics, i.e p50, p90, trimmean (the one Zain suggests), if we ever want to use this script manually